source/crud/crud.rst

```diff
@@ -445,7 +445,7 @@ Read
     maxAwaitTimeMS: Optional<Int64>;
 
     /**
-     * Maximum number of documents or index keys to scan when executing the query.
+     * Deprecated:4.0.Maximum number of documents or index keys to scan when executing the query.
```

Comment:
This formatting could be improved with some whitespace after the colon and period.

That said, how about adding a `@deprecated MongoDB 4.0` annotation below the existing description (above or below `@see` annotations)? Alternatively, just add "Deprecated: 4.0" as its own paragraph after the description, like we do for "Note:". Personally, I think `@deprecated` stands out the most when scanning these doc blocks.

Reply:
Should I do both? Deprecate in the description and @deprecate below @see?

Reply:
Note: Wrap `@works` with backticks in GitHub to avoid pinging users.

I think `@deprecated` (note the "d") above or below `@see` is preferable to adding this to descriptions paragraphs. I wouldn't advocate for both.